mergify_engine/context.py

```diff
@@ -1446,6 +1446,9 @@ def get_depends_on(self) -> typing.List[github_types.GitHubPullRequestNumber]:
     async def update_pull_check_runs(
```

Comment:
This method as a really weird name. I had to read the code to understand what it does because I don't think the name describes it correctly 😢

Reply:
I proposed something else.